components/safe_builtins/renderer/safe_builtins_helpers.h

```diff
@@ -23,7 +24,7 @@ namespace brave {
 v8::MaybeLocal<v8::Value> LoadScriptWithSafeBuiltins(
     blink::WebLocalFrame* web_frame,
     const std::string& script,
-    const std::string& name);
+    const absl::optional<std::string>& name);
```

Comment:
let's move this to different PR, we hide all the usage together if we want to hide script names.
Also this is not required for implementing eip 6963 since you didn't add additional script to run and implement it using v8 code eventually .